dockstore-webservice/src/main/java/org/kohsuke/github/authorization/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *    Copyright 2023 OICR
```

Comment:
missing UCSC

Reply:
note for self for Monday: I think I copied this from somewhere, so update that too

Reply:
> note for self for Monday: I think I copied this from somewhere, so update that too

In case you don't know, IntellIJ has a copyright profile it can use to auto insert the copyright into new files. And it takes parameters too, so you don't have to update it every year -- mine starts off with `Copyright $today.year OICR and UCSC`